pyproject.toml

```diff
@@ -186,7 +184,13 @@ clickhouse = ["clickhouse-connect", "sqlalchemy"]
 dask = ["dask", "regex"]
 datafusion = ["datafusion"]
 druid = ["pydruid", "sqlalchemy"]
-duckdb = ["duckdb", "duckdb-engine", "sqlalchemy", "sqlalchemy-views"]
+duckdb = [
+  "duckdb",
+  "duckdb-engine",
+  "packaging",
+  "sqlalchemy",
+  "sqlalchemy-views",
```

Comment:
These changes look like they are from a pre-7.0 commit. I think you need to rebase.

Reply:
Thanks for pointing that out, I think I screwed this one up while resolving conflicts during the previous rebase.